src/utils/ota/version.go

```diff
@@ -2,7 +2,7 @@ package ota
 
 var (
 	// Version is a releae version embedded into the app
-	Version = "v0.0.1"
+	Version = "development"
```

Comment:
@arfgdev can you ensure OTA updates functionality works well with this change?

Reply:
+

Reply:
it does not work 😅
`Binary version validation failed: Invalid character(s) found in major number "development"`

Reply:
It won't work because it doesn't follow the semver semantics

Reply:
I'll close the PR then